### PR TITLE
fixed the bug Text in 'AboutUs' page disappears in desktop mode

### LINKED ISSUE #162: Bug: Text in 'AboutUs' page disappears in desktop mode
**Describe the bug**

In desktop mode, some lines of description in the AboutUs page below 'How We Help' are currently not visible.

What it looks like currently:
after_fork

What it should look like (Expected Behaviour):
before_fork

Additionally, in mobile view, only one expected line of text appears:
after_fork_mobile

**To Reproduce**
1. Go to 'AboutUs' page
4. Notice that two lines of the subheading are missing.

**Expected behavior**

Expected text (below the heading) must be displayed properly in all modes, for all screen sizes.

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Chrome

**Additional Context**
The required text is already included in the aboutstyle.js file, it just needs to be made visible.

### PR DESCRIPTION
**Pull request by satavshreya**
#### Description
Issue FIXES #162 

//MADE CSS CHANGES://
.topline{
  display: flex;
  justify-content: center;
  font-weight: bold;
  margin-top: 0px;
  padding-top: 20px;
  color: #A0616A;
  text-transform: uppercase;
  background-color: #181818;
}
.subline{
  display: flex;
  margin-top: -18px;
  padding-top: 20px;
  justify-content: center;
  color: white;
  background-color: #181818;
  
}

#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
NA

#### Checklist

- [ ] I have added my name in the contributors list at the end of README.md file.
- [x ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

 The title of my pull request is a short description of the requested changes.
- [x] Open Source Program names(GirlScript Summer of Code 2021)

### CODE REVIEW

**Comment by satavshreya** [comment]
@khushishikhu @Aaishpra please review this one

File: periodo/components/about/aboutstyle.module.css [hunk 1]
```diff
@@ -85,7 +86,7 @@
   z-index: 2;
 }
 .container{
-    position: relative;
+    position:relative ;
```
Comment (Aaishpra) [inline periodo/components/about/aboutstyle.module.css:89]: why did you remove space from here? please add it back

File: periodo/components/about/aboutstyle.module.css [hunk 2]
```diff
@@ -10,31 +10,32 @@
...
   color: #A0616A;
   text-transform: uppercase;
   background-color: #181818;
 }
 .subline{
   display: flex;
-  margin-top: -20px;
+  margin-top: -18px;
   padding-top: 20px;
-  padding-bottom: 20px;
```
Comment (Aaishpra) [inline periodo/components/about/aboutstyle.module.css:33]: Was removing this necessary

File: periodo/components/about/aboutstyle.module.css [hunk 3]
```diff
@@ -10,31 +10,32 @@
...
-  padding-left: 10px;
```
Comment (Aaishpra) [inline periodo/components/about/aboutstyle.module.css:34]: here too same as above

**Review by Aaishpra — CHANGES_REQUESTED** [review]
Please check my feedback below

**Comment by satavshreya** [comment]
@Aaishpra I have made the respective changes now what has to be the next step do i have to make a new pull request? or just push changes into my local newbranch?

**Comment by Aaishpra** [comment]
Just push commits in this PR only no need for new PR

**Comment by satavshreya** [comment]
@Aaishpra  pushed all of it Do let me know

**Review by khushishikhu — APPROVED** [review]
@satavshreya LGTM!! Thank you for your contribution.

**Comment by satavshreya** [comment]
Happy to contribute to this :) Want to keep on contributing and really shape Periodo as a best project!

**Pull request merged**